e2e_test/e2e_test.go

```diff
@@ -495,6 +620,17 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 					return err != nil && errors.IsNotFound(err)
 				}, testutils.TestAPITimeout*30*2, testutils.TestRetryInterval).Should(BeTrue(), "Pod %s was not fully deleted in time", podName)
 			}
+
+			for i := 0; i < sfcNum; i++ {
+				namespace, sfcName1, nfName1 := sfcGetName(i)
+				dpuSideClient.Delete(context.TODO(), sfcCreate(namespace, sfcName1, nfName1))
```

Comment:
Deleting the SFC should cause the pods to be eventually deleted. There is no need to manually delete the pod